documentation/project-proposal.md

```diff
@@ -113,11 +113,11 @@ Then I should be able to filter the expenses by category and the list would upda
 
 ### Budget Management
 
-1. As a user, I want to define weekly, monthly, and annual budgets so that I can set spending limits for different time frames.\
+1. As a user, I want to define weekly, monthly, and annual budgets for specific categories so that I can set spending limits for different time frames.\
```

Comment:
```suggestion
1. As a user, I want to define a budget with frequency (weekly/monthly/annual) and a category so that I can set spending limits for different time frames and categories.\
```